src/api/posts/get/getPosts.js

```diff
@@ -0,0 +1,13 @@
+import axios, * as others from 'axios'
+
+export async function getAllPosts () {
+  const SERVER_URL = 'http://localhost:3000/posts'
+
+  const data = axios.get(SERVER_URL).then(Response => {
+    if (Response.data) {
+      return Response.data
+    }
+  })
+
+  return data
+}
```

Comment:
Try to simplify your method logic, I suppose it is a better idea to handle errors like this 'chaining' way:

const data = axios
  .get(SERVER_URL)
  .then(response => {
     console.log(data)
     return response.data
  })
  .catch(error => {
     console.log(error)
     // OR whatever error object STRUCTURE is  console.log(error.response.data.error)
  })